src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -84,6 +109,32 @@ protected Invitation()
         public DateTime? ModifiedAtUtc { get; private set; }
         public int? ModifiedById { get; private set; }
 
+        private void AddDpScope(IList<McPkg> mcPkgs)
+        {
+            if (Type != DisciplineType.DP || mcPkgs.Count == 0 || _commPkgs.Any())
```

Comment:
@janid1967 this will never throw exception as the code is now, because it is caught in constructor or in Edit(), but should I leave it so that it will catch it if someone makes changes in the future? (sam for AddMdpScope)

Reply:
I'll investigate

Reply:
removed